js/scroll.js

```diff
@@ -1,18 +1,47 @@
-const SCROLL_DELAY = 3000;
-var scrollDistance;
-var intervalScroll;
+const SCROLL_PLAN_DELAY = 3000, SCROLL_INFO_DELAY = 3000;
+var intervalScrollPlan, intervalScrollInfo;
+var infoContainer = document.getElementById('info-container');
```

Comment:
yes. The difference is: 

    var infoContainer = document.getElementById('info-container');

can only be executed when the document is created to a point where "info-container" is present.

Putting it in the function, allows the JavaScript code to run though at any time, regardless of the state of the document. Sometimes, when the code is executed, the document contains no elements. Therefore, we have `document.load` or `document.addEventListener("load", ... )`